### ISSUE TITLE
Can't upload new sequences with upload script

### ISSUE THREAD

**Issue by ToeBee** [openstreetcam/openstreetview.org#127]
For a few days now I have been unable to upload any new images using the `upload_photos_by_exif.py` script. The server first returns a HTTP 500 error and then when the script attempts to retry, it rejects the image as a duplicate (HTTP 400 with apiCode 660 returned) and goes into an infinite retry loop (see https://github.com/openstreetcam/upload-scripts/issues/21)

I have deleted the `sequence_file.txt` from the directory repeatedly. The script does write out a new sequence file with a new number in it each time I try to upload. However the `count_file.txt` is never created because no images ever get successfully uploaded.

My profile now says I have 7 tracks that are in the middle of uploading. Because of a recent change to the profile page that no longer lets me see sequences that are not fully processed, I can no longer delete partially uploaded sequences.

I just did a git pull to make sure I was running the latest version of the script so it seems like this must be a server side error (hence not opening this issue on the upload scripts repo) and I'm wondering if this is somehow specific to my account. I'm not sure how many of the other users active in the last few days have been using the script vs the app.

**Comment by ToeBee** [comment]
I thought maybe it was related to the access token so I deleted the access_token.txt file and tried re-authorizing it. This did not help either.

**Comment by bogdan-telenav** [comment]
Thanks for this. I will make some tests today 👍

**Comment by bogdan-telenav** [comment]
@ToeBee As you said this was a server side problem. Also this was fixed.
Thanks for reporting and mapping

**Comment by ToeBee** [comment]
I was indeed able to upload some sequences today. My user profile page says they have all been processed but I do not see them in my profile. And actually... I have several sequences that are not showing up on my profile page now. I think 7 to be exact. Basically everything I have uploaded since June 21st or 22nd is not reflected on my user profile. Although my score on the leaderboard is definitely going up - by more than it should I think. Like... a lot more. It says I have over 900,000 points for the week. I've uploaded less than a thousand pictures this week.
